rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -477,7 +477,11 @@ def try_all_rules(
             if err:
                 continue
 
-            if decoding_output.event is not None or len(decoding_output.action_items) > 0:
+            if (
+                decoding_output.event is not None or
+                len(decoding_output.action_items) > 0 or
+                decoding_output.process_swaps
```

Comment:
why this addition right now? I am not sure I get it.

Reply:
for the future reader: https://discord.com/channels/657906918408585217/1080783409640648765/1365308378502533181